src/Content/Text/BBCode.php

```diff
@@ -1094,7 +1118,7 @@ function (array $attributes) {
 	 */
 	public static function removeSharedData(string $body): string
```

Comment:
At least this seems like a good place for unit tests
input = `$body`
expected = return value